dockerclient.go

```diff
@@ -26,7 +27,7 @@ const (
 	// (v1.17) and
 	// ListVolumes, {Remove,Create}Volume, ListNetworks,
 	// {Inspect,Create,Connect,Disconnect,Remove}Network (v1.21)
-	APIVersion = "v1.15"
+	APIVersion = "v1.20"
```

Comment:
Ya unfortunately I don't think copy to container functionality was added then.  Although I think we still send the request so perhaps I can just leave it 1.15 for now.